cinema_service/settings.py

```diff
@@ -134,7 +139,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
Code quality issue: The MEDIA_ROOT setting is set to an absolute path. It's a better practice to use a path relative to your project's base directory.